lib/spack/spack/package.py

```diff
@@ -2666,6 +2666,10 @@ class Package(PackageBase):
     # packages, if need be
     run_after('install')(PackageBase.sanity_check_prefix)
 
+    spack.directives.variant('spack_built_type', default='rel_with_deb',
+                             help="Set to control compiler flags for optimization \
```

Comment:
Add `values=spack.build_types.build_types.values()`, so we don't have to validate elsewhere.

Reply:
All set!